### PR TITLE
Restyle the admin to look more like the designs

### PR DESCRIPTION
**Pull request by hammerfunk**

### CODE REVIEW

File: app/assets/javascripts/admin/common.js.coffee [hunk 1]
```diff
@@ -0,0 +1,19 @@
+ready = ->
+  updateCollapseHeaderClass()
+
+  $(document).on "click", ".panel-title > a", (e) ->
+    e.preventDefault()
+    _this = $(this)
+
+    setTimeout(updateCollapseHeaderClass, 400)
+
+updateCollapseHeaderClass = () ->
```
Comment (matthewford) [inline app/assets/javascripts/admin/common.js.coffee]: Empty parameter list is forbidden

File: app/assets/javascripts/admin/common.js.coffee [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  $(".panel-title > a").each () ->
```
Comment (matthewford) [inline app/assets/javascripts/admin/common.js.coffee]: Empty parameter list is forbidden

File: app/assets/stylesheets/admin/base.scss [hunk 3]
```diff
@@ -1,6 +1,53 @@
...
+    vertical-align: middle;
+  }
+
+  > tbody > tr {
+    > td,
+    > th {
+      background-color: #fff;
+    }
+  }
+}
```
Comment (matthewford) [inline app/assets/stylesheets/admin/base.scss]: Rule declaration should be followed by an empty line

File: app/assets/stylesheets/admin/base.scss [hunk 4]
```diff
@@ -163,36 +226,54 @@
     }
   }
 
-// Search
-  .search-input {
+  .icon-role {
```
Comment (matthewford) [inline app/assets/stylesheets/admin/base.scss]: Line should be indented 0 spaces, but was indented 2 spaces

File: app/assets/stylesheets/admin/base.scss [hunk 5]
```diff
@@ -1,6 +1,53 @@
 @import "mixins";
 
 // Basic
+.ellipsis {
+  display: inline-block;
+  max-width: 100%;
+  white-space: nowrap;
```
Comment (matthewford) [inline app/assets/stylesheets/admin/base.scss]: Properties should be ordered display, max-width, text-overflow, white-space, overflow

File: app/assets/stylesheets/admin/base.scss [hunk 6]
```diff
@@ -163,36 +226,54 @@
...
+    font-size: 16px;
+    font-style: italic;
+  }
 
-      @include is-retina {
-        background-image: image-url("icon-search@2.png");
-      }
+  input[type="submit"] {
+    position: absolute;
+    width: 30px;
```
Comment (matthewford) [inline app/assets/stylesheets/admin/base.scss]: Properties should be ordered position, top, right, bottom, width, margin, padding, border, background, background-image, background-repeat, background-position, background-size, text-align, text-indent

File: app/assets/stylesheets/admin/base.scss [hunk 7]
```diff
@@ -376,252 +486,368 @@
...
+  }
+}
 
-    .comment,
-    .comment-footer {
-      padding: 15px 10px;
-    }
+.comments-container {
+  display: none;
+  color: #fff;
```
Comment (matthewford) [inline app/assets/stylesheets/admin/base.scss]: Properties should be ordered display, background, color

File: app/assets/stylesheets/admin/base.scss [hunk 8]
```diff
@@ -376,252 +486,368 @@
...
-      top: 50%;
-      right: 10px;
-      margin-top: -5px;
-      background-image: image-url("icon-close-comment.png");
-      background-size: 10px 10px;
-      background-repeat: no-repeat;
-      background-position: right;
+  .icon-delete-comment {
+    position: absolute;
+    width: 10px;
```
Comment (matthewford) [inline app/assets/stylesheets/admin/base.scss]: Properties should be ordered position, top, right, width, height, margin-top, background-image, background-repeat, background-position, background-size

File: app/assets/stylesheets/admin/base.scss [hunk 9]
```diff
@@ -376,252 +486,368 @@
...
-        background-image: image-url("icon-close-comment@2.png");
-      }
+    @include is-retina {
+      background-image: image-url("icon-close-comment@2.png");
     }
   }
+}
+
+.overlay-delete {
+  position: absolute;
```
Comment (matthewford) [inline app/assets/stylesheets/admin/base.scss]: Properties should be ordered display, position, top, right, bottom, left, width, background, text-align

File: app/assets/stylesheets/admin/base.scss [hunk 10]
```diff
@@ -163,36 +226,54 @@
...
-      padding: 0;
-      text-align: left;
-      text-indent: -9999px;
-      background: transparent;
-      background-repeat: no-repeat;
-      background-position: center center;
-      background-image: image-url("icon-search.png");
-      background-size: 14px 19px;
-      border: none;
+  input[type="search"] {
```
Comment (matthewford) [inline app/assets/stylesheets/admin/base.scss]: Avoid qualifying attribute selectors with an element.

File: app/assets/stylesheets/admin/base.scss [hunk 11]
```diff
@@ -163,36 +226,54 @@
...
+    padding-right: 40px;
+    font-size: 16px;
+    font-style: italic;
+  }
 
-      @include is-retina {
-        background-image: image-url("icon-search@2.png");
-      }
+  input[type="submit"] {
```
Comment (matthewford) [inline app/assets/stylesheets/admin/base.scss]: Avoid qualifying attribute selectors with an element.

File: app/assets/stylesheets/admin/base.scss [hunk 12]
```diff
@@ -259,62 +340,85 @@
...
 
 // Sortable Tables
-  th.sortable {
-    a {
-      padding-right: 16px;
-      color: #333;
-      background-repeat: no-repeat;
-      background-position: right center;
-      background-size: 10px 6px;
+th.sortable {
```
Comment (matthewford) [inline app/assets/stylesheets/admin/base.scss]: Avoid qualifying class selectors with an element.

File: app/assets/stylesheets/admin/base.scss [hunk 13]
```diff
@@ -259,62 +340,85 @@
...
+}
 
-        &.ordered-desc {
-          background-image: image-url("icon-arrow-sort-up.png");
-
-          @include is-retina {
-            background-image: image-url("icon-arrow-sort-up@2.png");
-          }
+// Filterable Dropdowns
+th.filter {
```
Comment (matthewford) [inline app/assets/stylesheets/admin/base.scss]: Avoid qualifying class selectors with an element.

File: app/assets/stylesheets/admin/base.scss [hunk 14]
```diff
@@ -376,252 +486,368 @@
...
+.overlay-delete {
+  position: absolute;
+  display: none;
+  width: 100%;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  text-align: center;
+  background: rgba(0,0,0,0.8);
```
Comment (matthewford) [inline app/assets/stylesheets/admin/base.scss]: Commas in function arguments should be followed by a single space

File: app/helpers/form_answer_helper.rb [hunk 15]
```diff
@@ -11,10 +11,13 @@ def application_flags(form_answer)
...
   def application_comments(form_answer)
-    output = ""
+    output = "<span class='icon-comment'>Comments: <span class='comment-count'>"
     if form_answer.comments.any?
-      output += "<span class='icon-comment'>Comments: <span class='comment-count'>#{form_answer.comments.size}</span></span>"
+      output += "#{form_answer.comments.size}"
+    else
+      output += "0"
     end
+    output += "</span></span>";
```
Comment (matthewford) [inline app/helpers/form_answer_helper.rb]: Do not use semicolons to terminate expressions.

File: app/assets/stylesheets/admin/base.scss (hunk 3 above)
Comment (hammerfunk) [inline app/assets/stylesheets/admin/base.scss]: It does have an empty line ... :unamused:

**Comment by mauricioac** [comment]
Looks good, just waiting on travis

**Pull request merged**